app/models/document_type/title_and_base_path_field.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class DocumentType::TitleAndBasePathField
+  def id
+    "title_and_base_path"
```

Comment:
Have you considered something like `self.class.name.demodulize.underscore` so that we don't have to repeat it for each field type? Though we'd need a parent field class that both of these inherit from.

Reply:
Indeed I did, although I didn't think it would be that easy! While it's tempting to change it here, since it would enforce consistency, I'd prefer to do that in a subsequent PR, along with a few other refactorings. I'll make a note...